.github/std-unexploded.yml

```diff
@@ -106,3 +106,48 @@ jobs:
       - uses: divnix/std-action/run@main
         with: {ffBuildInstructions: true, remoteStore: "ssh-ng://eu.nixbuild.net"}
 
+  deploy-to-eu:
+    <<: *job
+    needs: [discover, images]
+    name: ${{ matrix.target.jobName }} (eu-central-1)
+    env:
+      AWS_REGION: eu-central-1
+    permissions:
+      id-token: write
+      contents: read
+      deployments: write
+    environment:
+      name: dev-preview
+      url: https://backend.dev-preview.eks.lw.iog.io
+    if: fromJSON(needs.discover.outputs.hits).deployments.apply != '{}'
+    strategy:
+      matrix:
+        target: ${{ fromJSON(needs.discover.outputs.hits).deployments.apply }}
+    steps:
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@main
+        with:
+          role-to-assume: ${{ env.AWS_ROLE_ARN }}
+          aws-region: ${{ env.AWS_REGION }}
+
+      - uses: nixbuild/nix-quick-install-action@v25
```

Comment:
So nixbuild is working well now? Cross-referencing with the other PR where removing Nixbuild seemed to result in a significant build time improvement.